.travis.yml

```diff
@@ -0,0 +1,35 @@
+services:
+- docker
+
+before-install:
+  - echo "$DOCKER_PASSWORD" | docker login --username $DOCKER_USERNAME --password-stdin #login to docker hub
+  - curl https://cli-assets.heroku.com/install.sh | sh  #install heroku
+  - docker login --username=$HEROKU_LOGIN --password=$HEROKU_API_KEY registry.heroku.com  #login to registry.heroku.com
```

Comment:
You should also use the `echo ... | docker login ... --password-stdin` option here, like your first docker login command.